### PR TITLE
Implement AddGelu and SkipLayerNorm

### PR DESCRIPTION
**Pull request by yufenglee**
**Description**: Describe your changes.

**Motivation and Context**
- Why is this change required? What problem does it solve?
- If it fixes an open issue, please link to the issue here.

### CODE REVIEW

File: onnxruntime/contrib_ops/cpu/bert/bias_gelu_fusion.cc [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+  const Tensor* B = ctx->Input<Tensor>(1);
+  const auto bias_dims = B->Shape().GetDims();
+  if (bias_dims.size() != 1) {
+    return ORT_MAKE_STATUS(ONNXRUNTIME, INVALID_ARGUMENT,
+                           "Input 1 is expected to have 1 dimensions, got ", bias_dims.size());
+  }
+
+  int64_t bias_len = bias_dims[0];
+  if (bias_len != input_dims[input_dims.size() - 1]) {
```
Comment (tianleiwu) [inline onnxruntime/contrib_ops/cpu/bert/bias_gelu_fusion.cc]: check input_dims.size() >= 1

File: onnxruntime/contrib_ops/cpu/bert/bias_gelu_fusion.cc [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    BiasGelu,
+    kMSDomain,
+    1,
+    kCpuExecutionProvider,
+    KernelDefBuilder().TypeConstraint("T", DataTypeImpl::GetTensorType<float>()),
+    BiasGelu<float>);
+
+template <typename T>
+static void ComputeOneTask(int64_t task_idx,
+                          int64_t bias_len,
```
Comment (tianleiwu) [inline onnxruntime/contrib_ops/cpu/bert/bias_gelu_fusion.cc]: alignment

File: onnxruntime/contrib_ops/cpu/skip_layer_norm.h [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+#include "core/framework/tensor.h"
+
+namespace onnxruntime {
+namespace contrib {
+
+template <typename T>
+class SkipLayerNorm final : public OpKernel {
+ public:
+    SkipLayerNorm(const OpKernelInfo& op_kernel_info);
+  Status Compute(OpKernelContext* p_op_kernel_context) const override;
```
Comment (tianleiwu) [inline onnxruntime/contrib_ops/cpu/skip_layer_norm.h]: indent

File: onnxruntime/contrib_ops/cpu/skip_layer_norm.cc [hunk 4]
```diff
@@ -120,53 +86,41 @@ Status SkipLayerNorm<T>::Compute(OpKernelContext* p_ctx) const {
...
+                                                   if (nullptr != bias_data) {
+                                                     value += bias_data[h];
+                                                   }
+                                                   p_output[h] = value;
+                                                   mean += value;
+                                                   mean_square += value * value;
+                                                 }
+
+                                                 mean = mean / hidden_size;
+                                                 mean_square = sqrt(mean_square / hidden_size - mean * mean + float(1e-12));
```
Comment (fs-eire) [inline onnxruntime/contrib_ops/cpu/skip_layer_norm.cc]: static_cast<T>(1e-12)

File: onnxruntime/contrib_ops/cpu/bert/bias_gelu_fusion.cc [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+  T* tmp_data = static_cast<T*>(temp_data_buf_ptr.get());
+
+  const T* X_data = X->template Data<T>();
+  const T* B_data = B->template Data<T>();
+  T* Y_data = Y->template MutableData<T>();
+  int64_t task_count = X->Shape().Size() / bias_len;
+
+  concurrency::ThreadPool::TryBatchParallelFor(ctx->GetOperatorThreadPool(),
+                                               static_cast<int32_t>(task_count),
+                                               [=](int32_t task_idx) {
```
Comment (fs-eire) [inline onnxruntime/contrib_ops/cpu/bert/bias_gelu_fusion.cc]: use reference capture as we will always execute the function body inside the lifecycle

File: onnxruntime/contrib_ops/cpu/bert/bias_gelu_fusion.cc [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+                                                 const T* p_input = X_data + task_idx * bias_len;
```
Comment (fs-eire) [inline onnxruntime/contrib_ops/cpu/bert/bias_gelu_fusion.cc]: please try to adjust the indent of function body. see https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/contrib_ops/cpu/bert/embed_layer_norm.cc#L80

**Pull request merged**